.github/actions/generate-docs/action.yml

```diff
@@ -0,0 +1,75 @@
+name: 'Generate and Deploy Documentation'
+description: 'Generate and deploy documentation files.'
+inputs:
+  tembo_repository:
+    description: 'GitHub repository to work with for documentation generation.'
+    required: true
+  website_repository:
+    description: 'Website repository to update with documentation.'
+    required: true
+  ssh_key:
+    description: 'SSH key for repository access.'
+    required: true
+  tembo_branch:
+    description: 'Branch to checkout for documentation generation.'
+    required: true
+  website_branch:
+    description: 'Branch to update with generated documentation.'
+    required: true
+
+on:
+  push:
+    paths:
+      - 'tembo-cli/**'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'tembo-cli/**'
+    branches:
+      - main
+      - develop
+      - feature/*
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out the tembo repo
+      uses: actions/checkout@v4
+      with:
+        repository: ${{ inputs.tembo_repository }}
+        ssh-key: ${{ inputs.ssh_key }}
+        ref: ${{ inputs.tembo_branch }}
+        path: 'tembo-repo'
+
+    - name: Generate Documentation
+      run: |
+        cd tembo-repo/tembo-cli
+        pwd
+        ls -lah
+        cargo run -- --markdown-help > ../command-reference.md
+      shell: bash
+
+    - name: Check out the website repo
+      uses: actions/checkout@v4
+      with:
+        repository: ${{ inputs.website_repository }}
+        ssh-key: ${{ inputs.ssh_key }}
+        ref: main
+        path: 'website-repo'
+
+    - name: Copy documentation to website repository
+      run: |
+        mkdir -p website-repo/src/content/docs/development/cli/
+        cp tembo-repo/command-reference.md website-repo/src/content/docs/development/cli/
+      shell: bash
+
+    - name: Commit and push documentation to website repository
+      run: |
+        cd website-repo
+        git config user.name "coredb-service-user"
+        git config user.email "admin@github.com"
+        git add src/content/docs/development/cli/command-reference.md
+        git checkout -b ${{ inputs.website_branch }}
+        git commit -m "Update command reference documentation" && git push origin ${{ inputs.website_branch }} || echo "No change in docs!"
```

Comment:
Can you please include the short git commit hash of the tembo CLI commit in the commit message, like this.

This part https://github.com/tembo-io/tembo/blob/c6e96c80d27f5dc119260c671061e6f915de36ed/.github/workflows/deploy.yml#L23

should run on the code checkout of the CLI

Then you have the short SHA in an "output" of a step. Then you can include it like this

```
git commit -m "Update command reference documentation: ${{ steps.versions.outputs.SHORT_SHA }}" && git push origin ${{ inputs.website_branch }} || echo "No change in docs!" 
```